package.json

```diff
@@ -10,19 +13,29 @@
   },
   "dependencies": {
     "@next/font": "13.1.6",
-    "@types/node": "18.13.0",
+    "@prisma/client": "^4.10.1",
     "@types/react": "18.0.28",
     "@types/react-dom": "18.0.11",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
     "eslint": "8.34.0",
     "eslint-config-next": "13.1.6",
+    "fastify": "^4.13.0",
```

Comment:
why is fastify installed here? Don't we use next api endpoints?

Reply:
removed